addons/website_slides/wizard/slide_channel_invite.py

```diff
@@ -95,8 +95,8 @@ def action_invite(self):
 
     def _prepare_mail_values(self, slide_channel_partner):
         """ Create mail specific for recipient """
-        subject = self.env['mail.template']._render_template(self.subject, 'slide.channel.partner', slide_channel_partner.id, post_process=True)
-        body = self.env['mail.template']._render_template(self.body, 'slide.channel.partner', slide_channel_partner.id, post_process=True)
+        subject = self.env['mail.template']._render_template(self.subject, 'slide.channel.partner', slide_channel_partner.ids, post_process=True)[slide_channel_partner.id]
+        body = self.env['mail.template']._render_template(self.body, 'slide.channel.partner', slide_channel_partner.ids, post_process=True)[slide_channel_partner.id]
```

Comment:
Because invite is not a template. _render_field is available only for template model itself; here we are rendering a custom string. It may come from a template, but is not the template anymore (can be customized in the wizard).